web/angular-wallet/src/locales/bg.json

```diff
@@ -49,6 +49,7 @@
   "all_blocks": "Всички блокове",
   "all_transactions": "Всички сделки",
   "amount": "Сума",
+  "amount_nqt": "Количество",
```

Comment:
damn here are some translations from another context.... well, nevermind...if all is merged  it's gonna be ok